app/pb_kits/playbook/pb_progress_simple/_progress_simple.jsx

```diff
@@ -1,21 +1,74 @@
+/* @flow */
 import React from 'react'
-import PropTypes from 'prop-types'
+import classnames from 'classnames'
+import { buildCss } from '../utilities/props'
 
-const propTypes = {
-  className: PropTypes.string,
-  id: PropTypes.string,
+type ProgressSimpleProps = {
+  align?: "left" | "center" | "right",
+  className?: String | Array<String>,
+  dark?: Boolean,
+  data?: String,
+  id?: String,
+  max: String,
+  muted: Boolean,
+  percent: String,
+  value: Number,
+  width: String
 }
 
-class ProgressSimple extends React.Component {
-  render() {
-    return (
-      <div className="pb_progress_simple">
-        <span>{'PROGRESS SIMPLE CONTENT'}</span>
-      </div>
-    )
+const ProgressSimple = ({
+  align,
+  className,
+  dark = false,
+  max,
```

Comment:
On any of the props that are required (they don't have `?`) we will want to ensure some default is set